README.md

```diff
@@ -32,125 +32,149 @@ mapbox-gl-style-build
-
- Supports the following operations: 
- - `*`: multiplication
- - `/`: division
- - `+`: addition
- - `-`: subtraction
-
- Also added support for passing optional options object to round the modified value in different ways:
- - `floor`: boolean
- - `ceil`: boolean
- - `round`: boolean
- - `toFixed`: number
-
- _Example:_
- ```js
-  // style-template.js
-  const { mergeVariables, modifyNumberVariables } = require('mapbox-gl-style-build');
-
-  const textSizes = require('../variables/textSizes');
-
-  module.exports.context = {
-    textSizes: modifyNumberVariables(textSizes, '*', 2, { round: true }),
-    ...
-  };
- ```
+Takes a variable or object specifying variables and applies a math function to the values. Expression values have the math function applied to all outputs within the expression.
+
+Supports the following operations:
+
+- `*`: multiplication
+- `/`: division
+- `+`: addition
+- `-`: subtraction
+
+Also added support for passing optional options object to round the modified value in different ways:
+
+- `floor`: boolean
+- `ceil`: boolean
+- `round`: boolean
+- `toFixed`: number
+
+_Example:_
+
+```js
+ // style-template.js
+ const { mergeVariables, modifyNumberVariables } = require('mapbox-gl-style-build');
+
+ const textSizes = require('../variables/textSizes');
+
+ module.exports.context = {
+   textSizes: modifyNumberVariables(textSizes, '*', 2, { round: true }),
+   ...
+ };
+```
 
 ## Development
 
- 1. Clone this rep
- 2. `yarn install`
- 3. `yarn watch`
- 4. Edit files in `src/` and they will be built into `dist/`
+1.  Clone this rep
+2.  `yarn install`
+3.  `yarn watch`
+4.  Edit files in `src/` and they will be built into `dist/`
 
 ## Implementation
 
-Implementation of the build system is most easily done with a combination of a custom script and manual work.
+Implementation of the build system is most easily done with a combination of a provided script and manual work.
+
+Based on the prerequisite decisions about primary differentiators for variants, this script can do the initial work of:
 
-Based on the prerequisite decisions about primary differentiators for variants, a single use script can do the initial work of: 
 - Breaking existing styles out into the style template and layer JS files
 - Imposing a file structure on the repo
 
+```bash
+create-layer-templates
+    --in-dir=styles
+    --out-dir=templates
+    --base-path=styles/base-style.json
+```
+
+The parameters are as follows:
+
+- `--in-dir`: the style directory containing existing variant styles to break out into template files
+- `--out-dir`: the directory to build your template files to
+- `--base-path`: the name of the style in the "in-dir" that is the base style
+
+After running the implementation script, you may manually delete the initial styles directory unless reusing it for building the templates to. The script avoids this step as there may occasionally be reason to leave these pre-build-system styles for posterity.
+
```

Comment:
This implementation section is all that changed in the README. Everything else is a formatting change from prettier. Let me know if we want to leave this as is, remove the extra formatting, or make a followup issue to add prettier to this repo.